Docs/source/usage/parameters.rst

```diff
@@ -204,6 +204,7 @@ Domain Boundary Conditions
     Options are:
 
     * ``Periodic``: This option can be used to set periodic domain boundaries. Note that if the fields for lo in a certain dimension are set to periodic, then the corresponding upper boundary must also be set to periodic. If particle boundaries are not specified in the input file, then particles boundaries by default will be set to periodic. If particles boundaries are specified, then they must be set to periodic corresponding to the periodic field boundaries.
+    * ``pec``: This option can be used to add a Perfect Electric Conductor at the simulation boundary. If ``warpx.do_electrostatic = labframe`` is also specified the boundary potentials can also be set through ``boundary.potential_lo_x/y/z`` and ``boundary.potential_hi_x/y/z`` (default `0`).
```

Comment:
@roelof-groenewald Could you clarify whether this will work also for ``warpx.do_electrostatic = relativistic``?

Reply:
All the functionality to set the BC is called through `computePhi`, which is called for both the LabFrame and Relativistic solve cases, so yes the BC values will be set in either case. I'll update the documentation.